manifests/b/bmatzelle/Gow/0.8.0/bmatzelle.Gow.locale.en-US.yaml

```diff
@@ -1,26 +1,26 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
-
-PackageIdentifier: bmatzelle.Gow
-PackageVersion: 0.8.0
-PackageLocale: en-US
-Publisher: bmatzelle
-# PublisherUrl: 
-# PublisherSupportUrl: 
-# PrivacyUrl: 
-# Author: 
-PackageName: Gow
-PackageUrl: https://github.com/bmatzelle/gow
-License: Copyright (c) 2006-2010 Brent Matzelle
-LicenseUrl: https://github.com/bmatzelle/gow/blob/master/licenses/Gow-License.txt
-# Copyright: 
-# CopyrightUrl: 
-ShortDescription: Gow (Gnu On Windows) is the lightweight alternative to Cygwin.
-# Description: 
-# Moniker: 
-# Tags: 
-# Agreements: 
-# ReleaseNotes: 
-# ReleaseNotesUrl: 
-ManifestType: defaultLocale
-ManifestVersion: 1.1.0
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: bmatzelle.Gow
+PackageVersion: 0.8.0
+PackageLocale: en-US
+Publisher: bmatzelle
+PublisherUrl: https://www.linkedin.com/in/bmatzelle

```

Comment:
```suggestion
```

This is the URL that the publisher has on their GitHub profile but we can omit it here to fix the error